### DISCUSSION TITLE
my manta token deduct my wallet during contract intraction

### DISCUSSION THREAD

**Discussion by Srb6123** [blockscout/blockscout#9430] [Support] [answered]
my manta token deduct my meamask wallet from manta pacifice chain,my contract inraction fail not rever my toknen in my wallet my fail trancation id is -0x0802a62e57b13de522b050bf049b114e10d861cc7db455dd01eff2f2baf9d8e2, my contract intraction on manta pacific network

**Comment by Srb6123** [comment]
plz help

**Comment by Srb6123** [comment]
0x0802a62e57b13de522b050bf049b114e10d861cc7db455dd01eff2f2baf9d8e2
Details
Token transfers
Internal txns
Logs
State
Raw trace
Transaction hash

0x0802a62e57b13de522b050bf049b114e10d861cc7db455dd01eff2f2baf9d8e2
Status and method

Failed
send
Revert reason

Raw:
0x08c379a00000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000002645524332303a207472616e7366657220616d6f756e7420657863656564732062616c616e63650000000000000000000000000000000000000000000000000000
Decoded:
�y� &ERC20: transfer amount exceeds balance
Block

[1419568](https://pacific-explorer.manta.network/block/1419568)
|
2064 Block confirmations
Timestamp

6h ago
|
Feb 20 2024 14:30:39 PM (+05:30 UTC)
|
Confirmed within <= 10 secs
From

[0x503E7EfD2BdE7eE223ED1Bcd34639AF366615AdA](https://pacific-explorer.manta.network/address/0x503E7EfD2BdE7eE223ED1Bcd34639AF366615AdA)
Interacted with contract

[Bridge](https://pacific-explorer.manta.network/address/0x9B36f165baB9ebe611d491180418d8De4b8f3a1f)
Value

0 ETH

($0)
Transaction fee

0.000048674338475407 ETH

($0.14427171272787585614)
Gas price

0.000000000000100074 ETH
(0.000100074 Gwei)
Gas usage & limit by txn

76,507
|
47,500,000
0.16%
Gas fees (Gwei)

Base: 0.000000074|
Max: 0.000100231|
Max priority: 0.0001
L1 gas used by txn

4,308

L1 gas price

0.000000033890481045 ETH

(33.890481045 Gwei)

L1 fee

0.000048666682113889 ETH

($0.14424901911920927378)
L1 fee scalar

0.333333

**Comment by Srb6123** [comment]
how long time to revert my toke in my wallet

**Comment by sl1depengwyn** [comment, marked as answer]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
